### ISSUE TITLE
Support validation of vocabulary contexts without workspace

### ISSUE THREAD

**Issue by karelklima** [opendata-mvcr/sgov#137]
There is a workspace validation endpoint (/[workspaceFragment]/validate) that validates data integrity of a workspace. It is used by OntoGrapher to tell user whether there are any issues with the data or not. https://github.com/opendata-mvcr/sgov/blob/main/src/main/java/com/github/sgov/server/controller/WorkspaceController.java#L307

We need to migrate to per-vocabulary validation, since we're dropping workspaces. The validation process should be roughly the same where applicable, it should be possible to reuse most of the logic already implemented.

We might still use the workspace validation feature before publishing the workspace, but in addition to the existing endpoint we need a new `/validate/{vocabularyFragment}` endpoint that would handle validation of that particular vocabulary context.

To discuss: it may be useful to create an endpoint to validate multiple vocabularies, e.g. `/validate?vocabulary=XXXX&vocabulary=YYYY`. It shouldn't bee too much extra work on backend and it will be super useful for OntoGrapher.

**Comment by blcham** [comment]
- implementation of the validation algorithm is straightforward 
  - currently, validation of workspace is a validation of all its vocabularies separately minimal work is needed here
- providing REST API of this kind is trivial too:
  - `/workspace/[wsFragment]/validate/{vocabularyFragment}`
  - `/workspace/[wsFragment]/validate?vocabulary=XXXX&vocabulary=YYYY`

-----
Here I am not sure **yet**, that it is really sufficient and we don't need to provide API as follows:
  -  `/vocabularies/[vocFragment]/validate`  // this would be quite big challenge to refactor

**Comment by karelklima** [comment]
@blcham Well, OntoGrapher is not going to know anything about a workspace. So we need to be able to pass just a list of vocabulary IRIs to a validate endpoint. OntoGrapher is currently validating the whole workspace, we should not change that.

As I said, we need a `/validate?vocabulary=XXX&vocabulary=YYY` endpoint. Not tied to a workspace in any way. The arguments should be IMHO whole IRIs, not just fragments.

**Comment by blcham** [comment]
1)  my misunderstanding came from the fact that I did not believe that you meant `/validate/{vocabularyFragment}` as an absolute path of REST API endpoint. IMHO, it is not very restful, so I assume that you wanted to continue on existing path which was `/workspace/[wsFragment]/validate`. At the same time it did not makes sense to me much so I wrote.

So I suggest the API `/vocabularies/[vocFragment]/validate` and `/vocabularies/validate?vocabulary=XXXX&vocabulary=YYYY`.

2) Moreover, If I understand it correctly, by `vocabulary=XXXX` you mean vocabulary context, so instead it should be 
 `vocabularyContext=XXXX` right?  So should I rather have parameter `vocabularyContext` or the trick here is that vocabularyContext==vocabulary, so before the task we need to migrate data so it holds?
